navigator_engine/tests/unit_tests/test_graph_loader.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import networkx
+import pickle
+from navigator_engine import model
+from navigator_engine.common.graph_loader import graph_loader
+from navigator_engine.tests.util import app
+
+
+@pytest.mark.usefixtures('with_app_context')
```

Comment:
Since you are using the app context, this file should probably sit under the "integration tests" folder.

Unit tests would mock everything out that is needed to only test the logic of individual functions. This makes them super fast to run.  

The tests below require a working DB instance, and a flask app to work, so you are testing that many bits of code work together.  I think it is fine to just move the file over to the integration tests folder.